server/src/main/java/org/opensearch/extensions/ExtensionsOrchestrator.java

```diff
@@ -71,6 +71,7 @@ public class ExtensionsOrchestrator implements ReportingService<PluginsAndModule
     public static final String REQUEST_EXTENSION_LOCAL_NODE = "internal:discovery/localnode";
     public static final String REQUEST_EXTENSION_CLUSTER_SETTINGS = "internal:discovery/clustersettings";
     public static final String REQUEST_EXTENSION_REGISTER_REST_ACTIONS = "internal:discovery/registerrestactions";
+    public static final String REQUEST_EXTENSION_REGISTER_TRANSPORT_ACTIONS = "internal:discovery/registertransportactions";
```

Comment:
Are we having separate action for extensions?

Reply:
There is an open issue: https://github.com/opensearch-project/OpenSearch/issues/4334. Lets do a sweep all together in a dedicated PR.